.azure/pipelines/jobs/default-build.yml

```diff
@@ -107,13 +106,17 @@ jobs:
     displayName: Install Node 10.x
     inputs:
       versionSpec: 10.x
-  - ${{ if and(eq(parameters.agentOs, 'Windows'), eq(parameters.codeSign, 'true')) }}:
+  - ${{ if eq(parameters.agentOs, 'Windows') }}:
+    - powershell: ./eng/scripts/InstallJdk.ps1 '11.0.1'
+      displayName: Install JDK 11
+  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.agentOs, 'Windows'), eq(parameters.codeSign, 'true')) }}:
     - task: MicroBuildSigningPlugin@1
       displayName: Install MicroBuild Signing plugin
       condition: and(succeeded(), in(variables['_SignType'], 'test', 'real'))
       inputs:
         signType: $(_SignType)
         zipSources: false
+        feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
```

Comment:
Doubt this is harmful but why is it necessary now?

Reply:
Different agents have different versions of the MicroBuildSigning Plugin. The new agents we are switching to require us to specify this parameter.